src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/configuration/configuration.component.html

```diff
@@ -18,6 +18,10 @@
         <option *ngFor="let opt of filter.options">{{ opt }}</option>
       </select>
     </div>
+    <button type="button"
+            class="btn btn-primary btn-sm"
+            (click)="resetFilter()"
+            i18n>Reset Filter</button>
```

Comment:
IMHO the refresh icon would be more appropriate here. I know there is a refresh icon on this table but on most tables except some, the refresh icon is omitted as the tables auto refresh every 5 seconds, which is the case for the configuration page, too.

So I'm suggestion to move the refresh icon at the end to reset the filter instead of using manual refresh and the reset filter button.

Reply:
I'm not sure if I get you right (please correct me if not), but I won't use the refresh button to refresh the table AND reset the filter. Also I'm not in favor of moving the refresh button to a different position than on any other table.

Reply:
From a UX perspective I'd avoid using the refresh button to reset the filter.

To reset all filters with one click I'd add something like the following behind the filter dropdowns:

```html
<div class="form-group filter">
    <a (click)="resetFilter()">
        <i class="fa fa-filter fa-2x"></i>
        <i class="fa fa-times fa-1x" style="margin-left: -10px"></i>
    </a>
</div>
```

Preview: https://codepen.io/anon/pen/vvvvxq

Reply:
@Devp00l , there is also a reset icon https://fontawesome.com/icons/undo, which can be used, what do you think ?

Reply:
According https://ux.stackexchange.com/questions/61707/whats-the-best-option-to-reset-a-filter a button would be appropriate.

Reply:
`undo` looks good to me or the solution without a button from @Exotelis. I don't think a button will look good here in our case, because buttons only show up on the left in a listing in our dashboard.